.github/workflows/deploy.yml

```
@@ -0,0 +1,37 @@
+name: Deploy SACODE Application
+
+on:
+  pull_request:
+    branches: [develop]
```

Acho que seria interessante mantermos isso só na main, não?

```suggestion
    branches: [main]
```